src/xdp/trn_transit_xdp.c

```diff
@@ -559,14 +262,6 @@ static __inline int trn_process_inner_arp(struct transit_packet *pkt)
 	epkey.tunip[2] = *tip;
 	ep = bpf_map_lookup_elem(&endpoints_map, &epkey);
 
-	/* Don't respond to arp if endpoint is not found, or it is local to host */
-	if (!ep || ep->hosted_iface != -1 ||
-	    pkt->inner_arp->ar_op != bpf_htons(ARPOP_REQUEST)) {
-		bpf_debug("[Transit:%d:0x%x] Bypass ARP handling\n", __LINE__,
-			  bpf_ntohl(pkt->itf_ipv4));
-		return trn_switch_handle_pkt(pkt, *sip, *tip, *sip);
-	}
-
```

Comment:
ARP responder logic should be kept. I'm working on this file as well so I can bring it back later

Reply:
Ok sounds good!